classes/class-wc-connect-settings-pages.php

```diff
@@ -97,13 +144,15 @@ public function page() {
 			$hide_save_button = true;
 
 			?>
-				<h2>
-					<?php _e( 'Account', 'woocommerce' ); ?>
-				</h2>
 				<div class="wc-connect-admin-container" id="<?php echo esc_attr( $root_view ) ?>"></div>
 			<?php
 		}
 
+		public function output_packages_screen() {
```

Comment:
I know "double tab-bars" are used widely in the WP-Admin, but I find them to be one of the worst UX patterns imaginable. Totally off-topic here, but I think we should avoid that.

If we are marketing Connect as "a bunch of features", just like Jetpack, I don't think we *need* to group their configuration inside a tab. Instead we should put them where they make more sense. For example, Payment Selection would be under the "Checkout" tab, and the Packages Manager under the "Shipping" tab.

As I said, totally off-topic, but maybe we should give this a thought? ( @kellychoffman )